### DISCUSSION TITLE
Use MQTT to receive RFID events

### DISCUSSION THREAD

**Discussion by ybloch** [AxisCommunications/discussions#38] [VAPIX]
Hello,
Is there a way to receive in the RFID tag reader notifications at events the ID of the tag in the payload?

I use a camera intercom of the type:
`AXIS A8207-VE Mk II Network Video Door Station`

**Comment by vivekatoffice** [comment]
Hi @ybloch ,
You may need to explore [MQTT Event Bridge](https://www.axis.com/vapix-library/subjects/T10175981/section/t10173845/display)
